platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -569,7 +521,7 @@ public void run() {
   private native void getOfflineRegionStatus(OfflineRegionStatusCallback callback);
 
   @Keep
-  private native void deleteOfflineRegion(boolean pack, OfflineRegionDeleteCallback callback);
+  private native void deleteOfflineRegion(OfflineRegionDeleteCallback callback);
```

Comment:
Same comment as [above](https://github.com/mapbox/mapbox-gl-native/pull/15967/files#r350525808), I think I'm missing an understanding of this proposal, since I think it's clearer to have `deleteOfflineRegion` and `nativeClearAmbientCache` accept a `boolean runPackDatabaseAutomatically`.

What I agree with is eliminating the `...AndSkipPackDatabase()` APIs.